### ISSUE TITLE
Submenus lack indentation on mobile

### ISSUE THREAD

**Issue by dominikh** [google/docsy#745]
Submenus aren't indented in the mobile version.

<img src="https://user-images.githubusercontent.com/39825/138942643-e80d9ef9-cbf5-4ec9-a859-4a370ca3fc40.png" width=250>

This is problematic with a mix of submenus and leaves, because it's impossible to tell which leaves belong to a menu:
![image](https://user-images.githubusercontent.com/39825/138942987-7beb979d-b166-4d4f-892b-92f53456b15e.png)
vs
![image](https://user-images.githubusercontent.com/39825/138943032-d41f098f-c77d-4bec-9cb5-06166dc86f49.png)

**Comment by chalin** [comment]
You can adjust the indentation for mobile (and make it a little more compact on desktop), by using the following SCSS:

```scss
.td-sidebar-nav__section .ul-1 ul {
  padding-left: 1rem;
}
```

If you only want this on mobile then wrap the above using an appropriate `@include media-breakpoint-* { ... }`.

**Comment by chalin** [comment]
Of course, we could make the above style suggestion the docsy default. WDYT @LisaFC?

**Comment by narrenfrei** [comment]
> Hmm, yes, we've used Hugo content bundles for all our top level pages in the example site and user guide, so they come up as bold even if they're a single page (as you can see in your screenshot above: **Examples** is a single page), I can see how that would be more confusing if they weren't....
> 
> @narrenfrei you did a lot of "regular" sidebar work, any thoughts?

If I can remember right, the mobile navigation had also before my sidebar works no intention, but maybe I be wrong. I would also prefer a intention also on sidebar navigation.

@chalin's suggestion is right. At the moment we use 1.5em for padding-left on wide screens because we have so nice alignment also if someone uses the possibility of icons in the navigation. But in my opinion also 1em on all screen widths would be OK and maybe the better way.

**Comment by LisaFC** [comment]
Just revisiting this while going through old issues and actually do think a slight indent would work as default. @chalin, thoughts while you're deep in a giant pile of CSS?
